### PR TITLE
Codechef MARBLES code in C and Python with Readme (Issue #4683)

### PR DESCRIPTION
**Pull request by ka1shi**
**Fixes issue:**
#4683 

**Changes:**
Added C and python solution for Codechef Marbles (MARBLES) Problem.
Added Readme file for that problem.

### CODE REVIEW

File: code/online_challenges/src/codechef/MARBLES/MARBLES.c [hunk 1]
```diff
@@ -0,0 +1,39 @@
+#include <stdio.h>
```
Comment (arnavb) [inline code/online_challenges/src/codechef/MARBLES/MARBLES.c]: Reformat this file using clang-format and 4 spaces.

File: code/online_challenges/src/codechef/MARBLES/MARBLES.c [hunk 2]
```diff
@@ -0,0 +1,39 @@
...
+	        N=N-1; 
+	        k=k-1;
+	        ans=1;
+	        
+	        if(k>N/2)
+            k = N-k;
+            
+	        for(int i=0; i<k; i++)
+	        {
+	            ans = ans*(N-i);
```
Comment (arnavb) [inline code/online_challenges/src/codechef/MARBLES/MARBLES.c]: Use `*=`

File: code/online_challenges/src/codechef/MARBLES/MARBLES.c [hunk 3]
```diff
@@ -0,0 +1,39 @@
...
+	            ans = ans/(i+1);
```
Comment (arnavb) [inline code/online_challenges/src/codechef/MARBLES/MARBLES.c]: Use `/=`

File: code/online_challenges/src/codechef/MARBLES/MARBLES.py [hunk 4]
```diff
@@ -0,0 +1,24 @@
+T = int(raw_input());
+
+while(T>0):
```
Comment (arnavb) [inline code/online_challenges/src/codechef/MARBLES/MARBLES.py]: Reformat using black; port to Python 3; don't use unnecessary parentheses. All variables should be lowercase.

File: code/online_challenges/src/codechef/MARBLES/MARBLES.c [hunk 5]
```diff
@@ -0,0 +1,38 @@
...
+	long long int ans = 1;
+	
+	scanf("%d", &t);
+	
+	while(t > 0)
+	{
+	    t--;
+	    ans = 1;
+	    
+	    scanf("%d", &n);
```
Comment (arnavb) [inline code/online_challenges/src/codechef/MARBLES/MARBLES.c]: n and k can be declared above this line.

File: code/online_challenges/src/codechef/MARBLES/MARBLES.c [hunk 6]
```diff
@@ -0,0 +1,38 @@
...
+	
+   	int t, n, k;
+	long long int ans = 1;
+	
+	scanf("%d", &t);
+	
+	while(t > 0)
+	{
+	    t--;
+	    ans = 1;
```
Comment (arnavb) [inline code/online_challenges/src/codechef/MARBLES/MARBLES.c]: Just declare ans here.

File: code/online_challenges/src/codechef/MARBLES/MARBLES.c [hunk 7]
```diff
@@ -0,0 +1,38 @@
...
+	    else
+	    {
+	        n = n - 1; 
+	        k = k - 1;
+	        ans = 1;
+	        
+	        if(k > n/2)
+                k = n - k;
+            
+	        for(int i = 0; i < k; i++)
```
Comment (arnavb) [inline code/online_challenges/src/codechef/MARBLES/MARBLES.c]: Pre-increment in for loops.

File: code/online_challenges/src/codechef/MARBLES/MARBLES.py [hunk 8]
```diff
@@ -0,0 +1,21 @@
+t = int(input())
+
+while t > 0:
```
Comment (arnavb) [inline code/online_challenges/src/codechef/MARBLES/MARBLES.py]: If you haven't already, reformat this file using black.

File: code/online_challenges/src/codechef/MARBLES/MARBLES.py [hunk 9]
```diff
@@ -0,0 +1,21 @@
...
+    t -= 1
+    ans = 1
+
+    n, k = list(map(int, input().split()))
```
Comment (arnavb) [inline code/online_challenges/src/codechef/MARBLES/MARBLES.py]:
```suggestion
    n, k = map(int, input().split())
```

File: code/online_challenges/src/codechef/MARBLES/MARBLES.py [hunk 10]
```diff
@@ -0,0 +1,21 @@
...
+
+    if n == k:
+        print(ans)
+    else:
+        n = n - 1
+        k = k - 1
```
Comment (arnavb) [inline code/online_challenges/src/codechef/MARBLES/MARBLES.py]:
```suggestion
        n -= 1
        k -= 1
```

**Comment by ka1shi** [comment]
@arnavb  Kindly review it again. Thank You!

**Pull request merged**